src/pages/Information/Information.jsx

```diff
@@ -1,57 +1,81 @@
 import React from "react";
-import { FormattedMessage } from "react-intl";
+import Typography from "@mui/material/Typography";
+import { useIntl } from "react-intl";
 
-const Information = () => (
-  <>
-    <div data-test="information-display">
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-first-header"
-          defaultMessage="What is HacktoberFest?"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-first-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-      <img
-        data-test="image-display"
-        src="../src/assets/pumpkin_PNG86719.png"
-        alt="hacktoberfest"
-        style={{ height: "100px" }}
-      />
-    </div>
-    <div>
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-second-header"
-          defaultMessage="When & Where"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-second-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-    </div>
-    <div>
-      <h2 data-test="heading-display">
-        <FormattedMessage
-          id="Info-third-header"
-          defaultMessage="Come join the FUN!"
-        />
-      </h2>
-      <p data-test="para-display">
-        <FormattedMessage
-          id="Info-third-paragraph"
-          defaultMessage="Lorem ipsum..."
-        />
-      </p>
-    </div>
-  </>
-);
+const Information = () => {
+  const intl = useIntl();
+
+  const infoData = [
+    {
+      header: intl.formatMessage({
+        id: "Info-first-header",
+        values: { defaultMessage: "What is HacktoberFest?" },
+      }),
+      paragraph: intl.formatMessage({
+        id: "Info-first-paragraph",
+        values: { defaultMessage: "Lorem ipsum..." },
+      }),
+    },
+    {
+      header: intl.formatMessage({
+        id: "Info-second-header",
+        values: { defaultMessage: "When & Where" },
+      }),
+      paragraph: intl.formatMessage({
+        id: "Info-second-paragraph",
+        values: { defaultMessage: "Lorem ipsum..." },
+      }),
+    },
+    {
+      header: intl.formatMessage({
+        id: "Info-third-header",
+        values: { defaultMessage: "Come join the FUN!" },
+      }),
+      paragraph: intl.formatMessage({
+        id: "Info-third-paragraph",
+        values: { defaultMessage: "Lorem ipsum..." },
+      }),
+    },
+  ];
+
+  const displayInfo = () => {
+    let i = 0;
+    return (
+      <>
+        <div data-test="information-display">
+          {infoData.map(({ header, paragraph, defaultMessage }) => {
```

Comment:
🔧 I think `defaultMessage` isn't going to have a value here.

Reply:
Addressed:  62ea76e426f45d0cc1f295135945947f89142b64